android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -899,6 +903,33 @@ public void onReceivedHttpError(
       }
     }
 
+    @TargetApi(Build.VERSION_CODES.O)
+    @Override
+    public boolean onRenderProcessGone(WebView view, RenderProcessGoneDetail detail) {
+        // WebViewClient.onRenderProcessGone was added in O.
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
+            return false;
+        }
+        super.onRenderProcessGone(view, detail);
+
+        if(detail.didCrash()){
+          Log.e("RNCWebViewManager", "The WebView rendering process crashed.");
+        }
+        else{
+          Log.w("RNCWebViewManager", "The WebView rendering process was killed by the system.");
+        }
+
+        WritableMap event = Arguments.createMap();
```

Comment:
Why not use `createWebViewEvent(webView, webView.getUrl())`?

Reply:
You are right, I will look into it.